frontend/src/util/useVuetify.ts

```diff
@@ -1,9 +1,9 @@
 import { getCurrentInstance } from "vue";
```

Comment:
Der Code wird in den Klassen welchen diese Methode importieren nicht verwendet. Die Klasse kann somit entfernt werden. Falls diese doch Benötigt wird den Namen mit einem Großbuchstaben beginnen lassen und prägnanter benennen.

- src/components/zaehlstelle/charts/BelastungsplanKreuzungSvg.vue
- src/components/zaehlstelle/charts/StepLineChart.vue
- src/components/zaehlstelle/charts/ZeitreiheChart.vue